roles/user-setup/tasks/subtasks/setup-single-user.yml

```diff
@@ -13,17 +13,17 @@
   become: yes
   remote_user: root
   block:
-    - name: "create user {{ user.name }}"
-      user:
-        name: "{{ user.name }}"
+    - name: "create user {{ item.name }}"
```

Comment:
whenever we do an iteration across a tasks file, I actually prefer to have explicitly named loop variables so that there are no accidentaly collisions if we loop inside the task again. Also, "item" is just less self explanatory than "user"